data/data-pipeline/data_pipeline/etl/sources/michigan_ejscreen/etl.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.utils import get_module_logger
+from data_pipeline.score import field_names
+from data_pipeline.config import settings
+
+logger = get_module_logger(__name__)
+
+
+class MichiganEnviroScreenETL(ExtractTransformLoad):
+    """
```

Comment:
Nit: could you put a one-line summary on the first line? 

(Part of why we do this: then when you hover over the class in a lot of IDEs, the IDE will show you the one-line summary of the class.)

Reply:
I think I addressed what was sought - similar to what was asked in the other function methods etc: https://github.com/usds/justice40-tool/pull/1085#discussion_r776537538.